frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -2131,6 +2208,8 @@ function ReaderHighlight:onHighlightPress()
 end
 
 function ReaderHighlight:onStartHighlightIndicator()
+    -- disable that long-press icon, as, as NT device owners, we are slow people, and we are slow at selecting text. -- poire-z (2024)
```

Comment:
(That was a private joke, and was supposed to stay between you and me :))
Write something anyway to explain what/why we do that.